File: leetcode-challenges/fizzbuzz/haskell/fizzbuzz.hs
import System.Environment

main = do
   [divisorOne,divisorTwo,upperBound] <- map read<$>getArgs
   mapM putStrLn (fizz upperBound divisorOne divisorTwo)

fizz :: Int -> Int -> Int -> [String]
fizz n x y = 
  map fizzbuzz [1..n] where
  fizzbuzz i =
   if i `mod` x == 0 && i `mod` y == 0
    then "FizzBuzz"
   else if i `mod` x == 0
    then "Fizz"
   else if i `mod` y == 0
    then "Buzz"
   else
    show i

File: leetcode-challenges/fizzbuzz/javascript/fizzbuzz.js
var divisorOne = 3;
var divisorTwo = 5;
var upperBound = 100;

for(var i=1;i<=upperBound;i++){
  if((i%divisorOne == 0) && (i%divisorTwo == 0)){
    document.write("FizzBuzz");
  } else if (i%divisorOne == 0){
    document.write("Fizz");
  } else if (i%divisorTwo == 0){
    document.write("Buzz");
  } else {
    document.write(i);
  }
}